abdulfatai360/banka
# Fix failing tests

#### What does this PR do?
Fix failing tests

#### Description of Task to be completed?
- Have failing tests passed

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/58/head:fix-failing-test`
- Now that you have a copy of the branch, switch to it using  `git checkout fix-failing-test`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine
- Type `npm run build` to run the build script and populate the `dist/` directory containing transpiled code
- Run `npm test` command to start the automated test
